recipes/sdl_mixer/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class SDLMixerConan(ConanFile):
+    name = "sdl_mixer"
+    description = "SDL_mixer is a sample multi-channel audio mixer library"
+    topics = ("sdl_mixer", "sdl2", "sdl", "mixer", "audio", "multimedia", "sound", "music")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libsdl.org/projects/SDL_mixer/"
+    license = "Zlib"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "cmd": [True, False],
+               "wav": [True, False],
+               "flac": [True, False],
+               "mpg123": [True, False],
+               "mad": [True, False],
+               "ogg": [True, False],
+               "opus": [True, False],
+               "mikmod": [True, False],
+               "modplug": [True, False],
+               "fluidsynth": [True, False],
+               "nativemidi": [True, False],
+               "tinymidi": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "cmd": False,  # needs sys/wait.h
+                       "wav": True,
+                       "flac": True,
+                       "mpg123": False,
+                       "mad": False,
+                       "ogg": True,
+                       "opus": True,
+                       "mikmod": False,
+                       "modplug": False,
+                       "fluidsynth": False,
+                       "nativemidi": True,
+                       "tinymidi": False}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.settings.os != "Linux":
+            del self.options.tinymidi
+        else:
+            del self.options.nativemidi
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        self.requires("sdl/2.0.20")
+        if self.options.flac:
+            self.requires("flac/1.3.3")
+        if self.options.mpg123:
+            self.requires("mpg123/1.26") # TODO: this package is missing on the conan-center-index
+        if self.options.mad:
+            self.requires("libmad/0.15") # TODO: this package is missing on the conan-center-index
```

Comment:
```suggestion
            self.requires("libmad/0.15.1b")
```
https://github.com/conan-io/conan-center-index/tree/master/recipes/libmad